extensions/Particle3D/src/Particle3D/PU/PUMaterialManager.cpp

```diff
@@ -111,7 +111,7 @@ void PUMaterialCache::addMaterial(PUMaterial* material)
     {
         if (iter->name == material->name)
         {
-            AXLOG("warning: Material has existed (FilePath: %s,  MaterialName: %s)", material->fileName.c_str(),
+            AXLOGD("warning: Material has existed (FilePath: {},  MaterialName: {})", material->fileName.c_str(),
```

Comment:
.c_str() can be remove, and other location

Reply:
Ah thats the reason why this be not "changed"?
![image](https://github.com/axmolengine/axmol/assets/8652787/782e0db1-3db8-48e5-ad17-76a2600a7b77)